katas/src/lib.rs

```diff
@@ -28,13 +28,18 @@ pub fn check_solution(
     receiver: &mut impl Receiver,
 ) -> Result<bool, Vec<Error>> {
     let source_map = SourceMap::new(exercise_sources, Some(EXERCISE_ENTRY.into()));
+    let mut store = qsc::PackageStore::new(qsc::compile::core());
```

Comment:
After seeing this code repeated multiple times, I wonder if it would make sense to have utility functions like `qsc::PackageStore::with_core_lib` and `qsc::PackageStore::with_std_lib` to reduce the boilerplate code a bit.